docs/details/shards/GFX/BuiltinFeature.md

```diff
@@ -0,0 +1,3 @@
+- The `GFX.Feature` shard is used to create a custom feature using the parameters provided while the `GFX.BuilinFeature` shard is for adding a "ready-made" feature.
+
+- The feature object created by this shard is usually collected into a sequence, which is then in turn used to create a render pass using `GFX.DrawablePass`.
```

Comment:
There was some description about the built-in features at https://docs.fragcolor.com/reference/shards/shards/GFX/BuiltinFeature/
Would it be possible to add here? (or maybe on the enum page for BuiltinFeature)